sql/src/main/java/io/crate/planner/operators/LogicalPlan.java

```diff
@@ -208,6 +208,11 @@ default boolean preferShardProjections() {
 
     long numExpectedRows();
 
+    /**
+     * Returns an estimation of the size (in bytes) of each row returned by the plan
```

Comment:
Currently the implementations always calculate as if all columns are selected. "each row returned by the plan" indicates that this is based on the selected columns as defined in the plan.
I think we should clarify this. It shouldn't matter too much, since this is an estimate either way, but having comments that contradict the implementation just confuses readers.